vars/pushDockerImage.groovy

```diff
@@ -23,16 +23,13 @@ def call(body) {
 
         container(name: 'tools') {
             withCurrentRepo { def repoUrl, def repoName, def repoOwner, def repoBranch ->
-                def dockerImage = "${dockerRegistryURL}/${repoOwner.toLowerCase()}/${repoName.toLowerCase()}"
-                def dockerImageVersion = stakaterCommands.getBranchedVersion("${versionPrefix}.${env.BUILD_NUMBER}")
+                def imageName = repoName.split("dockerfile-").last().toLowerCase()    
+                def dockerImage = "${dockerRegistryURL}/${repoOwner.toLowerCase()}/${imageName}"
+                def dockerImageVersion = config.imageVersion ?: stakaterCommands.getBranchedVersion("${versionPrefix}.${env.BUILD_NUMBER}")
```

Comment:
How would you pass in the image version here?

Reply:
through config as parameter

Reply:
No, I mean will you hardcode the image version in jenkinsfile? Or read it from somewhere?

Reply:
currently I will hardcode but we can get this from a file, actually this is to create different tags of image, e.g. for centos7 I want to have stakater:base-centos:7 and similarly for other versions different tags

Reply:
alright